Mathlib/Tactic/CancelDenoms/Core.lean

```diff
@@ -149,7 +147,7 @@ partial def findCancelFactor (e : Expr) : ℕ × Tree ℕ :=
     | none => (1, .node 1 .nil .nil)
   | _ => (1, .node 1 .nil .nil)
 
-def synthesizeUsingNormNum (type : Expr) : MetaM Expr := do
+def synthesizeUsingNormNum (type : Q(Prop)) : MetaM Q($type) := do
   try
     synthesizeUsingTactic' type (← `(tactic| norm_num))
```

Comment:
Maybe not for this PR, but consider Qq-ifing `synthesizeUsingTactic'` too.

Reply:
Indeed, that already crossed my mind; though I figured that would have a larger fallout and was best left to a follow up.

Reply:
#9437